File: AdvantageFramework.Core/Database/Procedures/ClientPortalAlertView.cs
using System.Linq;

namespace AdvantageFramework.Core.Database.Procedures
{
    public static partial class ClientPortalAlertView
    {
        #region  Constants 



        #endregion

        #region  Enum 



        #endregion

        #region  Variables 



        #endregion

        #region  Properties 



        #endregion

        #region  Methods 

        public static System.Linq.IQueryable<Database.Views.ClientPortalAlertView> Load(Database.DbContext DbContext)
        {
            System.Linq.IQueryable<Database.Views.ClientPortalAlertView> LoadRet = default;
            LoadRet = from ClientPortalAlertView in DbContext.ClientPortalAlertView.AsQueryable()
                      select ClientPortalAlertView;
            return LoadRet;
        }

        #endregion
    }
}

File: AdvantageFramework/Database/Procedures/ClientPortalAlertView.vb
Namespace Database.Procedures.ClientPortalAlertView

    <HideModuleName()> _
    Public Module Methods

#Region " Constants "



#End Region

#Region " Enum "



#End Region

#Region " Variables "



#End Region

#Region " Properties "



#End Region

#Region " Methods "

        Public Function Load(ByVal DbContext As Database.DbContext) As System.Data.Entity.Infrastructure.DbQuery(Of Database.Views.ClientPortalAlertView)

            Load = From ClientPortalAlertView In DbContext.GetQuery(Of Database.Views.ClientPortalAlertView)
                   Select ClientPortalAlertView

        End Function

#End Region

    End Module

End Namespace
